wasmtime-api/tests/examples.rs

```diff
@@ -0,0 +1,38 @@
+use std::env;
+use std::process::{Command, Stdio};
+
+fn run_example(name: &'static str) {
+    let cargo = env::var("CARGO").unwrap_or("cargo".to_string());
+    let pkg_dir = env::var("CARGO_MANIFEST_DIR").unwrap_or(".".to_string());
```

Comment:
```suggestion
    let pkg_dir = env!("CARGO_MANIFEST_DIR");
```

I believe this is what is causing CI to fail.

Reply:
(That is, `CARGO_MANIFEST_DIR` set when cargo is compiling the crate, not when it is running tests)